homeassistant/components/twinkly/light.py

```diff
@@ -179,7 +207,19 @@ async def async_turn_on(self, **kwargs: Any) -> None:
                     await self._client.interview()
                     # Reagarrange from rgbw to wrgb
```

Comment:
let's fix this typo too 👍

Reply:
Just removed the comment.  It seems like it was a copy of the comment for rgbw.